js/utils/templateHelpers.js

```diff
@@ -0,0 +1,24 @@
+var app = require('../App.js').getApp(),
```

Comment:
This is good work. We could probably include polyglot in here, and I'm sure there are other places were we could change one-off solutions to common functions.